examples/reference/features/bokeh/Scalebar.ipynb

```diff
@@ -0,0 +1,116 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "df229fb9-0a56-47b4-a8d2-72cab8782624",
+   "metadata": {},
+   "source": [
+    "<div class=\"contentcontainer med left\" style=\"margin-left: -50px;\">\n",
+    "<dl class=\"dl-horizontal\">\n",
+    "  <dt>Title</dt> <dd> Scalebar</dd>\n",
+    "  <dt>Dependencies</dt> <dd>Bokeh</dd>\n",
+    "  <dt>Backends</dt> \n",
+    "    <dd><a href='./Scalebar.ipynb'>Bokeh</a></dd>\n",
+    "</dl>\n",
+    "</div>\n",
+    "\n",
+    "The `scalebar` feature adds an overlaid scalebar to the element. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1cee420f-3ad1-4b0c-ae66-b26d726d7a0a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import holoviews as hv\n",
+    "import numpy as np\n",
+    "\n",
+    "hv.extension(\"bokeh\")\n",
+    "\n",
+    "hv.RGB.load_image(\"../assets/pollen.png\").opts(scalebar=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "34200a10-f6d8-4ed5-a93b-9c62f0cbb7c2",
+   "metadata": {},
+   "source": [
+    "### Unit\n",
+    "The unit of the scalebar tries to infer it for the first `kdims`, and will fallback to `meter` if nothing is set. To overwrite this you can use `scalebar_unit`. This can ither be a string or a tuple with length of two - one for the dimension and one for the base-dimension. An example is shown below, where we have the dimension be `mm` and the base-dimension `m`. \n",
```

Comment:
```suggestion
    "The unit of the scalebar tries to infer it for the first `kdims`, and will fallback to `meter` if nothing is set. To overwrite this you can use `scalebar_unit`. This can either be a string or a tuple with length of two - one for the dimension and one for the base-dimension. An example is shown below, where we have the dimension be `mm` and the base-dimension `m`. \n",
```